### PR TITLE
cite: remove page from cite query in author profile and update limit

### PR DESCRIPTION
**Pull request by monaawi**
* Remove page from the cite query in the author profile
* Update cite limit to 1000
* INSPIR-2844

### CODE REVIEW

File: ui/src/literature/components/__tests__/CiteAllAction.test.jsx [hunk 1]
```diff
@@ -18,7 +18,7 @@ describe('CiteAllAction', () => {
 
   it('renders with disabled', () => {
     const wrapper = shallow(
-      <CiteAllAction numberOfResults={502} query={{ q: 'ac>2000' }} />
+      <CiteAllAction numberOfResults={1002} query={{ q: 'ac>2000' }} />
```
Comment (harunurhan) [inline ui/src/literature/components/__tests__/CiteAllAction.test.jsx]: `MAX_CITEABLE_RECORDS + 1`

File: ui/src/literature/components/CiteAllAction.jsx [hunk 2]
```diff
@@ -21,7 +22,12 @@ class CiteAllAction extends Component {
 
   async onCiteClick({ key }) {
     const { query } = this.props;
-    const queryString = stringify(query, { indices: false });
+    const citeQuery = {
+      // set default `sort` in case there is no `sort` in the query
+      sort: searchScopes.getIn(['literature', 'query', 'sort']),
+      ...omit(query, ['size', 'page']),
```
Comment (harunurhan) [inline ui/src/literature/components/CiteAllAction.jsx]: didn't find a test for `omit` and default `sort`.

File: ui/src/literature/components/__tests__/CiteAllAction.test.jsx (hunk 1 above)
Comment (michamos) [reply ui/src/literature/components/__tests__/CiteAllAction.test.jsx]: why the `+2`?

File: ui/src/literature/components/CiteModalAction.jsx [hunk 3]
```diff
@@ -9,6 +9,7 @@ import IconText from '../../common/components/IconText';
 import EventTracker from '../../common/components/EventTracker';
 import { CITE_FORMAT_OPTIONS, CITE_FORMAT_VALUES } from '../constants';
 import citeArticle from '../citeArticle';
+import { downloadFile } from '../utils';
```
Comment (harunurhan) [inline ui/src/literature/components/CiteModalAction.jsx]: let's move this common utils, and name it `downloadTextAsFile(text)`

**Pull request merged**